services/media/service/scripts/set-replication-slot-name.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-console */
+import hash from 'short-hash';
+import { updateEnvFile } from '../../../../scripts/helpers';
+
+async function main(): Promise<void> {
+  const slotName = `media_service_localization_slot_${hash(
+    new Date().toISOString(),
```

Comment:
I'm using current date/time as a unique value for the hash for the following reasons:
- I could use tenantId_environmentId, but then we need to re-adjust the order of steps, filling in all the values in root .env file first, and then running yarn db:reset. Currently the order is first we do apply-templates, then db:reset, then fill in custom .env values (e.g. tenant_id, environment_id)
- I could have used the developers machine name, but then if one would have multiple instances of media-service on one machine - conflicts could happen.

In the end, current datetime seemed like an appropriate choice because:
- this value is generated only for development
- the timestamp is hashed anyway (short-hash is already being used to generate test database names)
- the previous slot is deleted automatically on database reset

Reply:
> the previous slot is deleted automatically on database reset

This part is the important one to not run into too many slots issues.